exercises/concept/meltdown-mitigation/.docs/introduction.md

```diff
@@ -74,6 +74,22 @@ else:
 '13'
 ```
 
+Bear in mind that in the special case where a function returns a boolean, it should not use an `if`.
+Rather it should return the condition directly.
+
+```python
+# DO NOT use `if` and return `True`
+>>> def is_leap_year(year):
+        if (year % 4 == 0 and year % 100 != 0) or year % 400 == 0:
+            return True
+        else:
+            return False
+
+# instead return the condition directly
+>>> def is_leap_year(year):
+        return (year % 4 == 0 and year % 100 != 0) or year % 400 == 0
+```
+
```

Comment:
While this is excellent information to discuss, I don't think this exercise (_which is focused on conditionals_) is the place to do it.  This note sets an exception/rule about Booleans without going into the nuance that one can return an expression directly without assigning to an intermediary state variable -- which isn't just a rule around conditions and Booleans. I think it's better left to a mentor discussion/notes or explored in detail in an exercise on structuring functions/handling returns.

Reply:
I strongly disagree, unless someone can find a way to fix Task 1 so that it is not misleading (see [another discussion](https://github.com/exercism/python/pull/2566#discussion_r704648474)).

The learning exercise on Booleans (Ghost Gobble Arcade Game) already has students [returning expressions directly](https://github.com/exercism/python/blob/main/exercises/concept/ghost-gobble-arcade-game/.meta/exemplar.py#L43), and this concept is explained in the Basics exercise [Guido's Gorgeous Lasagna](https://github.com/exercism/python/blob/main/exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md).

Also, [knowing when not to use a tool is part of mastering that tool](https://twitter.com/kelseyhightower/status/963428093292457984?s=20).